### ISSUE TITLE
[BUG & REQUEST] About Mesh in v5 .

### ISSUE THREAD

**Issue by finscn** [pixijs/pixi.js#5645]
BUG:

When change the texture(with different uvs)  of SimpleRope & SimplePlane , 
the display will be wrong. 
e.g.
![image](https://user-images.githubusercontent.com/288367/57173997-1a662000-6e6a-11e9-94e0-36f61dc4ec2a.png)

the expectant result is  only one flight  (the middle one)

---------

REQUEST: 

1st:
  I hope SimpleRope & SimplePlane  could support anchor like  in v4 (not be merged , and it's created by  @ivanpopelyshev  ).
like this:  https://github.com/finscn/pixi.js/blob/v4-plus/src/mesh/Plane.js

2nd:
 I think the Rope should be a subclass of Plane,  because it's a specific plane, and there are some methods/properties could be shared by Rope & Plane.
like this: https://github.com/finscn/pixi.js/blob/v4-plus/src/mesh/Rope.js

In short,  the mesh of v5 is not as good as   the ivanpopelyshev-version mesh in v4.

**Comment by ivanpopelyshev** [comment]
I love you too. I'll port it soon :)
